youtube_dl/extractor/svt.py

```diff
@@ -141,21 +145,26 @@ class SVTPlayIE(SVTPlayBaseIE):
                     )
                     '''
     _TESTS = [{
-        'url': 'http://www.svtplay.se/video/5996901/flygplan-till-haile-selassie/flygplan-till-haile-selassie-2',
-        'md5': '2b6704fe4a28801e1a098bbf3c5ac611',
+        'url': 'https://www.svtplay.se/video/26194546/det-har-ar-himlen',
+        'md5': '2382036fd6f8c994856c323fe51c426e',
         'info_dict': {
-            'id': '5996901',
+            'id': 'jNwpV9P',
```

Comment:
This will break download archives.

Reply:
Archives as in playlist download? Did the id extraction change due to some earlier commit to svt.py?

Reply:
Looks like, it's off then.